src/components/WorkerStatusBadge.vue

```diff
@@ -0,0 +1,32 @@
+<template>
+  <template v-if="worker">
+    <p-tag class="worker-status-badge">
+      <WorkerStatusIcon :status="workerStatus" class="worker-status-badge__icon" />
+      {{ label }}
+    </p-tag>
+  </template>
+</template>
+
+<script lang="ts" setup>
+  import { computed } from 'vue'
+  import WorkerStatusIcon from '@/components/WorkerStatusIcon.vue'
+  import { WorkPoolWorker } from '@/models'
+  import { capitalize } from '@/utilities'
+
+  const props = defineProps<{
+    worker: WorkPoolWorker,
+  }>()
+
+  const workerStatus = computed(() => props.worker.status)
+
+  const label = computed(() => {
+    return capitalize(props.worker.status)
+  })
+</script>
+
+<style>
+.worker-status-badge { @apply
+  text-xs
+  pl-1
+}
```

Comment:
@brandonreid should we be customizing tag styles? I assume this is fine

Reply:
I removed the custom tag styling since it made the badges a little easier to read.